### ISSUE TITLE
Variable zabbix_agent_server do not work for zabbix_agent2

### ISSUE THREAD

**Issue by itdoginfo** [ansible-collections/community.zabbix#230]
##### SUMMARY
Variables **zabbix_agent_server**  and **zabbix_agent_serveractive** don't work for **zabbix_agent2: True**

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
community.zabbix.zabbix_agent
##### ANSIBLE VERSION

```paste below
ansible 2.9.12
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.13 (default, Aug 22 2020, 10:03:02) [GCC 6.3.0 20170516]

```

##### CONFIGURATION

```paste below
DEFAULT_EXECUTABLE(/etc/ansible/ansible.cfg) = /bin/bash
DEFAULT_SCP_IF_SSH(/etc/ansible/ansible.cfg) = True
DEFAULT_VAULT_PASSWORD_FILE(/etc/ansible/ansible.cfg) = /var/pass/ansible
HOST_KEY_CHECKING(/etc/ansible/ansible.cfg) = False
INTERPRETER_PYTHON(/etc/ansible/ansible.cfg) = auto_legacy_silent
RETRY_FILES_ENABLED(/etc/ansible/ansible.cfg) = False
TRANSFORM_INVALID_GROUP_CHARS(/etc/ansible/ansible.cfg) = ignore

```

##### OS / ENVIRONMENT / Zabbix Version
Ansible server: Debian 9.12
Ansible client: Debian 10.15
Zabbix Server 5.0.2

##### STEPS TO REPRODUCE

```yaml
- hosts: client

  remote_user: root

  become: yes
  become_user: root

  roles:
    - role: community.zabbix.zabbix_agent
      zabbix_agent2: True
      zabbix_agent_server: zabbix.example.com
      zabbix_agent_serveractive: zabbix.example.com
```

##### EXPECTED RESULTS

/etc/zabbix/zabbix_agent2.conf contains:
```
Server=zabbix.example.com
ServerActive=zabbix.example.com
```

##### ACTUAL RESULTS
/etc/zabbix/zabbix_agent2.conf contains:
```
Server=127.0.0.1
ServerActive=127.0.0.1
```
And zabbix_agentd.conf contains:
```
Server=zabbix.example.com
ServerActive=zabbix.example.com
```

For zabbix_agent2 using the same variables are used https://github.com/ansible-collections/community.zabbix/blob/main/roles/zabbix_agent/templates/zabbix_agent2.conf.j2

May be needs to specify something else in variables?

**Comment by dj-wasabi** [comment]
Hi @itdoginfo 

Can you check if this issue still arises? I think you installed the 1.0.0 version (And not from github/main) and there were some ossus fixed with the 1.1.0 release.

Kind regards,
Werner

**Comment by itdoginfo** [comment]
Hello!
Yes, in 1.1.0 everything worked for me. Thanks so much
